src/composables/getCollections.js

```diff
@@ -0,0 +1,30 @@
+import { ref } from 'vue';
+import { projectFirestore } from '../firebase/config';
+
+const getCollection = (collection) => {
+    const documents = ref(null);
+    const error = ref(null);
+
+    const collectionRef = projectFirestore
+        .collection(collection)
+        .orderBy('createdAt');
+
+    collectionRef.onSnapshot(
+        (snap) => {
```

Comment:
I think we should add new prop: uId to get documents that are belonged to only one patient.